elastic/kibana
# [Rollups] Add deprecated prompt

Closes https://github.com/elastic/kibana/issues/185861

## Summary

This PR replaces the empty prompt in Rollup Jobs with a deprecation prompt:

<img width="1494" alt="Screenshot 2024-06-10 at 17 55 38" src="https://github.com/elastic/kibana/assets/59341489/dd245836-6e9b-4efd-9657-74e2c372059f">

<br>
<br>

**How to test:**
1. Start Es and Kibana
2. Go to Stack Management -> Rollup Jobs
3. Since there are no existing rollup jobs, verify that the deprecation prompt is displayed and the Downsampling documentation link is correct.


### Checklist
